README.md

```diff
@@ -0,0 +1,60 @@
+#  NoiteDev
+[![Build Status](https://travis-ci.org/condessalovelace/mavenquickstart.svg?branch=master)](https://travis-ci.org/condessalovelace/mavenquickstart) ![Quality Gate](https://sonarcloud.io/api/project_badges/measure?project=br.com%3Amavenquickstart&metric=alert_status)
+
+## Description 
+
+This project illustrates an example of a simple registrer of students for learning.
+
+## Demostration
+
+![prototype](./img/prototype.png)
+
+## Starting
+
+To run the project, you will need to install the following programs:
+
+- [MySQL: a toolkit, necessary to carry out the project] (https://www.mysql.com/)
+- [XAMPP: a server to upload the database] (https://www.apachefriends.org/pt_br/index.html)
+- [Nodejs: Required to create the project] (https://nodejs.org/en/)
+- [VS Code: For project development] (https://code.visualstudio.com/)
+
+## Development
+
+To start the development, it is necessary to clone the GitHub project in a directory of your choice:
+
+```
+cd "Diretory of your preference"
+git clone https://github.com/Suspir0n/noiteDev.git
+```
```

Comment:
```suggestion
## ⌨️ Development

Use Gitpod, a free online dev environment for GitHub.

[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/Suspir0n/noiteDev.git)

Or use code locally using:
```
cd "Directory of your preference"
git clone https://github.com/Suspir0n/noiteDev.git
```
```